<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sistema de Puntaje</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_azul">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>786</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color:rgb(0, 0, 255);
}
</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="foto_azul">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>130</width>
            <height>150</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: 3px solid white;
border-radius: 8px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="tiempo_azul">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>230</y>
            <width>495</width>
            <height>120</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 0);
border: 3px solid black;
border-top-right-radius: 15px;
</string>
          </property>
          <widget class="QLabel" name="contador_1">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>10</y>
             <width>250</width>
             <height>90</height>
            </rect>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
font-size: 110px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="puntos_azul">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>25</y>
            <width>300</width>
            <height>240</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>160</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>11</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="info_azul">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>20</y>
            <width>540</width>
            <height>190</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="nombre_azul">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>400</width>
             <height>90</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>27</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Participante Azul</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="medico_azul">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>250</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Tiempo MÃ©dico</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QFrame" name="amonestaciones_azul">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>140</width>
             <height>50</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="amarilla_a1">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="amarilla_a2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="amarilla_a3">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </widget>
          <widget class="QFrame" name="bandera_azul">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>45</y>
             <width>130</width>
             <height>100</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;
border: 3px solid white;
border-radius: 8px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_rojo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	background-color: rgb(255, 0, 0);
	border-top: 3px solid black;	
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QFrame" name="tiempo_rojo">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-60</y>
            <width>495</width>
            <height>120</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 170, 0);
border: 3px solid black;
border-bottom-right-radius: 15px;</string>
          </property>
          <widget class="QLabel" name="contador_2">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>10</y>
             <width>250</width>
             <height>90</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
font-size: 110px;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>0:00</string>
           </property>
          </widget>
         </widget>
         <widget class="QFrame" name="foto_rojo">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>130</width>
            <height>150</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: 3px solid white;
border-radius: 8px;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="puntos_rojo1">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>25</y>
            <width>300</width>
            <height>240</height>
           </rect>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>160</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>false</kerning>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border: none;
color: rgb(255, 255, 255);
</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="text">
           <string>10</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QFrame" name="info_rojo">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>80</y>
            <width>540</width>
            <height>190</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background: transparent;
border: none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="nombre_rojo">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>400</width>
             <height>90</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>27</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Participante Rojo</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="medico_rojo">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>250</width>
             <height>50</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Tiempo MÃ©dico</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QFrame" name="amonestaciones_rojo">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>140</width>
             <height>50</height>
            </rect>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QFrame" name="amarilla_r1">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="amarilla_r2">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="amarilla_r3">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>10</y>
              <width>30</width>
              <height>30</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 239, 6);
border-color: rgb(0, 0, 0);
border: 2px solid black;
border-radius: 8px;</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </widget>
          <widget class="QFrame" name="bandera_rojo">
           <property name="geometry">
            <rect>
             <x>410</x>
             <y>45</y>
             <width>130</width>
             <height>100</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;
border: 3px solid white;
border-radius: 8px;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
